test/test_torch.py

```diff
@@ -3863,8 +3862,6 @@ def test_bernoulli_mem_overlap(self, device):
         p = torch.rand(6, device=device)
         with self.assertRaisesRegex(RuntimeError, 'unsupported operation'):
             x.bernoulli_(p=p)
```

Comment:
Hi, @kshitij12345!
Yes! I think two things are good to mention here:
1. OpInfo for `bernoulli` does not support `inplace_variant`, stating its inplace variant is different.
https://github.com/pytorch/pytorch/blob/0524b2829a1775016f8b7ee4db8205edebfc4520/torch/testing/_internal/common_methods_invocations.py#L13944-L13949
2. And inplace variant `bernoulli_` is a scalar and hence does not need an OpInfo, so no need for ErrorInputs.

Is this satisfactory? Thanks!

Reply:
I see. Plus the test only tests for functional variant. Thanks for clarifying

https://github.com/pytorch/pytorch/blob/96c8f6445926783fbf992b7e4be28a073d162557/test/test_ops.py#L222-L227

Reply:
Cool, I wasn't aware of this :) Thanks for the share!

Reply:
An interesting gap in the ErrorInput tests. Instead of just removing the FIXME comment lets update it to "FIXME: convert to ErrorInputs (but have to extend ErrorInputs to handle inplace-only errors!)".

We could do this by adding an option to the ErrorInput that specifies which variant to use in the test, for example.

Reply:
Thanks for the clear description, @mruberry! I've added the comments. And would give try to support in-place operations in ErrorInputs. Sounds interesting to me!!